src/lib/api/feed-manip.ts

```diff
@@ -25,7 +25,7 @@ export class FeedViewPostsSlice {
     if (this.items[0].reason?.indexedAt) {
       return this.items[0].reason.indexedAt as string
     }
-    return this.items[0].post.indexedAt
+    return this.items[this.items.length-1].post.indexedAt // ordering should depend on the most recent item in a slice
```

Comment:
not sure if there's another reason to order by the _head_ of a thread, but this definitely was preventing recent replies from appearing